.circleci/config.yml

```diff
@@ -670,7 +701,17 @@ workflows:
           force: true
           requires:
             - promote_chart_to_stable
+      - manual_release_notes_hold:
+          type: approval
+          context: CodacyDO
+          requires:
+            - tag_version_latest
```

Comment:
I have taken the liberty of adding this final manual step to ensure that the release notes are published ahead before creating the github release with a link to their page.